lessons/python/scikit-learn/UofTCoders_ML.ipynb

```diff
@@ -126,33 +121,17 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Feature Correlations\n",
-    "from scipy import stats\n",
-    "\n",
-    "feature_corr = pd.DataFrame(columns = iris['feature_names'])\n",
-    "for i in range(4):\n",
-    "    corr = []\n",
-    "    for j in range(4):\n",
-    "        corr.append(stats.pearsonr(iris.data[:, i], iris.data[:, j])[0])\n",
-    "    feature_corr.loc[i, ] = corr\n",
-    "feature_corr = feature_corr.rename(index = {0: iris['feature_names'][0],\n",
-    "                                            1: iris['feature_names'][1],\n",
-    "                                            2: iris['feature_names'][2],\n",
-    "                                            3: iris['feature_names'][3]})\n",
-    "feature_corr"
+    "iris_df.iloc[:,:4].corr()"
```

Comment:
Naming the column would make your intention more explicit:

```python
iris_df.drop('target', axis=1).corr()
```

Reply:
OK, I was curious why you did it "by hand"; I guess these are equivalent then.